pyconfig.h.in

```diff
@@ -914,8 +914,12 @@
 /* Define to 1 if you have the `sem_unlink' function. */
 #undef HAVE_SEM_UNLINK
 
+/* OSF1 documents and implements but does not declare sendfile,
+   and it does not match others. Skip it. */
+#ifndef __osf__
 /* Define to 1 if you have the `sendfile' function. */
 #undef HAVE_SENDFILE
+#endif
```

Comment:
I think this is ok, but if reviewers prefer, I could probably kinda autoconf it.

In particular, the check for sendfile would move out of the list of funcs, and into an individual call, that is conditional on not being OSF.
And/or we could confirm there is a declaration and what it is "like".
That would likely cleanup other systems.
Assuming the declaration is not difficult to attain.

I want to at least test the waters to see if anything is ok for OSF1, and if so, can refine it in this PR or a followup.